cmd/argo-watcher/argo.go

```diff
@@ -26,9 +25,20 @@ import (
 )
 
 var (
-	argoTimeout, _ = strconv.Atoi(os.Getenv("ARGO_TIMEOUT"))
-	config         = c.GetConfig()
-	retryAttempts  = uint((4 * (argoTimeout / 60)) + 1)
+	argoTimeout, _     = strconv.Atoi(os.Getenv("ARGO_TIMEOUT"))
```

Comment:
May be let's configure some fallback values? We can use GetEnv function from helpers instead of os.Getenv here.

Reply:
I agree. Used the helper, set default ARGO_TIMEOUT to 0, so that the configuration has backwards compatibility.